lib/src/kbasesearchengine/events/StatusEvent.java

```diff
@@ -33,6 +33,7 @@
     private final Optional<Integer> version;
```

Comment:
Since the CLI and event generator aren't covered by tests I assume they were tested manually?

Reply:
Yes. I ran both against CI and they look good.